src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -131,7 +86,6 @@ public static IAEItemStack loadItemStackFromNBT( final NBTTagCompound i )
 		}
 
 		final AEItemStack item = AEItemStack.create( itemstack );
-		// item.priority = i.getInteger( "Priority" );
 		item.setStackSize( i.getLong( "Cnt" ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "item" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2259)